code/mapping/ext_modules/mapping_common/entity.py

```
@@ -556,3 +605,82 @@ class ShapelyEntity:
+
+        Returns:
+            float: distance
+        """
+
+        return shapely.distance(self.poly, other.poly)
+
+
+def shape_debug_marker_array(
+    namespace: str,
+    timestamp: Optional[rospy.Time] = None,
+    lifetime: rospy.Duration = rospy.Duration.from_sec(0.5),
```

Ruff (0.8.2)

Do not perform function call `rospy.Duration.from_sec` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)